src/ansiblelint/cli.py

```diff
@@ -594,6 +594,11 @@ def get_config(arguments: list[str]) -> Options:
         msg = f"Failed to determine a valid project_dir: {options.project_dir}"
         raise RuntimeError(msg)
 
+    # expand user home dir in exclude_paths
+    options.exclude_paths = [
+        os.path.expandvars(os.path.expanduser(p)) for p in options.exclude_paths
```

Comment:
@ssbarnea I think we can use `Path.expanduser(p)`

Reply:
Not in this case, there is no expandvars inside pathlib and for these we need to stick to of paths as they are glob patterns, not real paths.